### PR TITLE
fix: checkbox make square smaller in order for browser to evenly move it from the center

### LINKED ISSUE #2152: checkbox tristate styling issue
**Describe the bug**

> Is this issue related to a specific component?
checkbox

> What version of the Fundamental Library Styles are you using?
latest version

**To Reproduce**
1. Go to [link](https://fundamental-styles.netlify.app/?path=/docs/components-forms-checkbox--desktop)

**Expected behavior**
the inside square is off
Screen Shot 2021-02-15 at 7 04 42 PM
Screen Shot 2021-02-15 at 7 04 55 PM
Screen Shot 2021-02-15 at 7 05 08 PM
Screen Shot 2021-02-15 at 7 05 21 PM

### PR DESCRIPTION
**Pull request by N1XUS**
## Related Issue
Closes #2152 

## Description
Make square smaller for compact checkbox, that way browser could evenly center it vertically and horizontally.

### Before:
image

### After:
image

#### Please check whether the PR fulfills the following requirements

1. The output matches the design specs
- [x] All values are in `rem`
- [x] Text elements follow the truncation rules
- [x] hover state of the element follow design spec
- [x] focus state of the element follow design spec
- [x] active state of the element follow design spec
- [x] selected state of the element follow design spec
- [x] selected hover state of the element follow design spec
- [x] pressed state of the element follow design spec
- [x] Responsiveness rules - the component has modifier classes for all breakpoints
- [x] Includes Compact/Cosy/Tablet design
- [x] RTL support
2. The code follows fundamental-styles code standards and style
- [x] only one top level `fd-*` class is used in the file
- [x] BEM naming convention is used
- [x] Mixins are used for repeatable code (`fd-rtl`, `fd-ellipsis`, `fd-flex`, `fd-selected`, `fd-focus`, ect.)
- [x] A11y support - keyboard support, screenreader support, proper ARIA attributes, etc.
- [x] `fd-reset()` mixin is applied to all elements
- [x] Variables are used, if some value is used more than twice.
- [x] Checked if current components can be reused, instead of having new code.
3. Testing
- [x] tested Storybook examples with "CSS Resources" `normalize` option 
- [x] tested Storybook examples with "CSS Resources" `unnormalize` option 
- [x] Verified all styles in IE11
- [x] Updated tests
- [x] last commit message should have `[ci visual]` so it can trigger chromatic visual regression (e.g. `test: run chromatic visual regression [ci visual]`)
4. Documentation
- [x] Storybook documentation has been created/updated
- [x] Breaking Changes [wiki](https://github.com/SAP/fundamental-styles/wiki/Breaking-Changes) has been updated in case of breaking changes.

### CODE REVIEW

File: src/checkbox.scss [hunk 1]
```diff
@@ -121,7 +121,8 @@ $block: #{$fd-namespace}-checkbox;
 
   &:indeterminate + .#{$block}__label--compact::before {
     content: "\e17b";
-    font-size: 0.5625rem;
+    font-size: 0.5rem;
+    padding: 0;
```
Comment (InnaAtanasova) [inline src/checkbox.scss:125]: Could you also fix the font-size of the icon in cozy (mobile) state? This is from the specs:
Screen Shot 2021-04-28 at 12 25 40 PM

Currently we have 0.8125rem and a padding. I think that if you change it to 0.75rem you. won't need a padding and inthiscase you won't need to add like 125 for compact mode.
Screen Shot 2021-04-28 at 12 30 54 PM

Comment (N1XUS) [reply src/checkbox.scss]: @InnaAtanasova Done. Please re-check.

**Review by InnaAtanasova — APPROVED** [review]

**Comment by InnaAtanasova** [comment]
@N1XUS you need a commit containing the text `[ci visual]` to trigger chromatic visual regression. Without it the checks won't pass.

File: src/checkbox.scss [hunk 2]
```diff
@@ -115,13 +115,12 @@ $block: #{$fd-namespace}-checkbox;
...
     content: "\e17b";
-    font-size: 0.8125rem;
-    padding-top: 0.0625rem;
+    font-size: 0.75rem;
   }
 
   &:indeterminate + .#{$block}__label--compact::before {
     content: "\e17b";
-    font-size: 0.5625rem;
+    font-size: 0.5rem;
```
Comment (droshev) [inline src/checkbox.scss:123]: i think if we make the font-size: 0.6rem the square will be more centered

**Review by droshev — CHANGES_REQUESTED** [review]
the compact square is still not centered and you need to run chromatic

**Review by droshev — APPROVED** [review]

**Pull request merged**